crowbar_framework/app/models/neutron_service.rb

```diff
@@ -277,6 +299,9 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
 
     network_nodes.each do |n|
       net_svc.allocate_ip "default", "public", "host",n
```

Comment:
`net_svc.allocate_ip "default", "public", "host",n`

can be removed, look on comment from:
https://github.com/crowbar/barclamp-neutron/pull/227/files#diff-1cab7f3b6842279ae4d6bd875e073cacL279